osgp/shared/osgp-dto/src/main/java/org/opensmartgridplatform/dto/valueobjects/smartmetering/ActualPowerQualityRequestDto.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright 2021 Alliander N.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.dto.valueobjects.smartmetering;
+
+public class ActualPowerQualityRequestDto implements ActionRequestDto {
+
+    private static final long serialVersionUID = 1544769605230783612L;
+
+    private final String profileType;
+
+    public ActualPowerQualityRequestDto() {
+        this(null);
```

Comment:
I wonder if this is appropriate.
I see one place where the profile type is used (in GetActualPowerQualityCommandExecutor) where a null value appears to result in `NullPointerException`.
I guess if `null` really is not an acceptable value this constructor may better apply some default (to be considered if some default would be appropriate, I think).

Reply:
This parameterless constructor is only used in test. Test was rewritten